ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -6,3 +6,9 @@ def det(input, name=None):
 
 
 det.unsupported_dtypes = ("float16", "bfloat16")
+
+
+def slogdet(input, name=None):
+    return ivy.slogdet(input)
+
+slogdet.unsupported_dtypes = ("float16", "bfloat16")
```

Comment:
This should be converted to a dictionary indicating the backend which doesn't support that particular dtype for your function, the [docs](https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html) have more details regarding the same.

Reply:
done